plugins/plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/json/NetworkSettings.java

```diff
@@ -17,7 +17,7 @@
 
 /** @author andrew00x */
 public class NetworkSettings {
-    private String   ipAddress;
+    private String   iPAddress;
```

Comment:
I agree with @benoitf that iPAddress is weird. Wouldn't it possible to use `@SerializedName` annotation instead?